resources/actions/check_nvbugs.py

```
@@ -0,0 +1,165 @@
+                   help="Numeric NVBugs ModuleInfo.Key to match against.")
+    g.add_argument("--module-name",
+                   help="NVBugs ModuleInfo.Value to match against.")
+    p.add_argument("--on-email-mismatch", choices=["warn", "fail"], default="warn")
+    p.add_argument("--on-bug-not-in-module", choices=["warn", "fail"], default="fail")
+    p.add_argument("--api-url", default=DEFAULT_API_URL)
+    return p.parse_args()
+
+
+def fetch_bug(api_url, bug_id, token):
+    url = f"{api_url.rstrip('/')}/api/Bug/GetBug/{bug_id}"
+    req = urllib.request.Request(url)
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)